src/types/index.ts

```diff
@@ -14,6 +14,7 @@ export type ContentPickerType =
 	| 'CONTENT_PAGE'
 	| 'COLLECTION'
 	| 'ITEM'
+	| 'BUNDLE'
 	| 'DROPDOWN'
 	| 'INTERNAL_LINK'
 	| 'EXTERNAL_LINK';
```

Comment:
Kun je hier ook nog even `SEARCH_QUERY` toevoegen, anders moet dit nog in aparte PR.